app/retail/templates/emails/grants/new_supporter.html

```diff
@@ -62,33 +67,43 @@
     color: white;
     border-radius: 2.7px;
     font-size: 17px;
-}
-#grow-oss {
+  }
+
+  #grow-oss {
     margin-top: 65px;
     margin-bottom: 30px;
-}
+  }
+
 </style>
-<img id="newSupporter" src="{% static "v2/images/newsupporter.png" %}" alt="New Supporter" title="New Supporter">
+<img id="newSupporter" src="{% static " v2/images/newsupporter.png " %}" alt="New Supporter" title="New Supporter">
 <h1 style="text-transform: none;">{% trans "You have a new supporter!" %}</h1>
 
 <hr>
 
-<div class="supporter"> <!-- Set {{github_username}} as variable for new supporter information -->
-    <a href="https://github.com/{{github_username}}"><img src="https://gitcoin.co/dynamic/avatar/{{github_username}}" width="50" height="50" /></a>
-    <br>
-    <a class="supporterName" href="https://github.com/{{github_username}}">{{github_username}}</a>
-    <h5 style="text-transform: none;">{%trans "User' has begun contributing" %}</h5> 
-    <h3 style="text-transform: none;">{{variable_amount_contribution}}</h3> <!-- Input variable amount here -->
+<div class="supporter">
+  <!-- Set {{ github_username }} as variable for new supporter information -->
+  <a href="https://github.com/{{ github_username }}">
+    <img src="https://gitcoin.co/dynamic/avatar/{{ github_username }}" width="50" height="50" />
+  </a>
+  <br>
+  <a class="supporterName" href="https://github.com/{{ github_username }}">{{ github_username }}</a>
+  <h5 style="text-transform: none;">{%trans "User' has begun contributing" %}</h5>
+  <h3 style="text-transform: none;">{{ variable_amount_contribution }}</h3>
+  <!-- Input variable amount here -->
 </div>
 
 <hr>
 
 <br>
-<p>{%trans "You can see a transaction history and withdraw funds" %} <a href="https://gitcoin.co/{{transaction_history}}">{%trans "here." %}</a></p> <!-- Input link to transaction history here -->
```

Comment:
Can we remove the references to "Withdraw". It is a user flow that we have taken out of the application.